app/views/layouts/application.haml

```diff
@@ -24,7 +24,8 @@
     - if(@view_type.eql?("map"))
       = javascript_include_tag "markerclusterer.js"
     - props = topbar_props
-    = react_component("TopbarApp", props: props, prerender: true)
+    - cache props do
```

Comment:
Doesn't this mean the props now contain information about the currently logged in user and that ends up in the cache + rendered at server side too? Wasn't the plan to only render the user agnostic version at server side and add with user rendering client side?